src/server/Document/Renderers/litePageTransforms/index.ts

```diff
@@ -0,0 +1,10 @@
+import transformAnchorTags from './transformAnchorTags';
+
+type Fn = (html: string) => string;
```

Comment:
Kinda related to below, would a generic type be useful here or not really? i.e

`type TransformFn<T> = (input: T) => T;`

so that if we have other functions which operate on strings and are transforming them but want to chain them together using your `pipe` function in the future it could be cool.

Maybe i'm not fully understanding the usage here though

Reply:
All good man, nice job!